webapi/src/main/java/com/dumbdogdiner/stickyapi/webapi/mojang/AshconResponse.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.webapi.mojang;
+
+import lombok.Getter;
+import lombok.SneakyThrows;
+import org.jetbrains.annotations.NotNull;
+
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.UUID;
+
+/**
+ * Package-local utility class to easily convert the received object from the Cached API to a java object.
+ */
+
+
+class AshconResponse {
```

Comment:
Internal classes in here are lacking javadoc!

Reply:
Why do internal data classes need javadoc?

Reply:
Because we still need to read this code when we're adding to it.

Reply:
> Why do internal data classes need javadoc?

We still need to understand and read the code.